e2e/README.md

```diff
@@ -0,0 +1,89 @@
+# About
+
+This folder contains an end to end testing suite written with playwright checking all of the main functionality provided by this template. Note it assumes you are using a postgres database on the backend and have configured a test database. This is required because the tests will **drop and recreate the test database** in order to ensure replicability between test runs.
+
+This test suite was built off of using the [medusa-starter-default](https://github.com/medusajs/medusa-starter-default) repository with the seed data from `data/seed.json`.
+
+# Setup
+
+## .env
+
+These tests have a number of dependent environment variables, with an example found in `.env.example`. You can setup your local environment by copying the example environment file
+
+```sh
+cp .env.example .env
```

Comment:
For me, the ENV variables needed to be added to the existing .env file in the project root. Declaring a separate .env in the e2e dir resulted in all variables being undefined.